src/backends/pyplot.jl

```diff
@@ -799,6 +799,9 @@ function py_set_scale(ax, sp::Subplot, axis::Axis)
     letter = axis[:letter]
     scale in supported_scales() || return @warn("Unhandled scale value in pyplot: $scale")
     func = getproperty(ax, Symbol("set_", letter, "scale"))
+    if PyPlot.version ≥ v"3.3" # https://matplotlib.org/3.3.0/api/api_changes.html
```

Comment:
Would you please send me the permalink to the title of this exact pyplot change?

Reply:
I could not find the relevant news myself

Reply:
Nevermind I found it now.